docs/README.md

```diff
@@ -572,9 +572,11 @@ The tables and functions in this schema store `pgroll`'s internal state and are
 $ pgroll start sql/03_add_column.json
 ```
 
-This starts the migration defined in the `sql/01_create_table.json` file.
+This starts the migration defined in the `sql/03_add_column.json` file.
 
-After starting a migration there will be two schema versions in the database; one for the old schema before the migration and one for the new version with the schema changes.
+After starting a migration there will be two schema versions in the database; one for the old schema before the migration (e.g. `public_02_create_table`) and one for the new version with the schema changes (e.g. `public_03_add_column`). At this point, the earlier version will be still available under the main schema name (e.g. `public`).
```

Comment:
I think it's important to make it clear that the versioned schema (like `public_02_create_table`) contain views on the tables in the `public` schema. Applications using schema managed by`pgroll` shouldn't connect directly to `public` but rather use the versioned views.

```suggestion
After starting a migration there will be two schema versions in the database; one for the old schema before the migration (e.g. `public_02_create_table`) and one for the new version with the schema changes (e.g. `public_03_add_column`).  Each schema contains views on the tables in the `public` schema.
```